doc/resources/jenkins/jenkins-sample.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```

Comment:
Can we reuse the yaml in doc/resources/fabric8?

Reply:
sadly not, since they use the variables define in maven but we use the environment variables injected by Jenkins.

Reply:
Can we just use their variable names? I compared the two files and there're some differences (like strategy), at least we need to be consistent.

Reply:
I'v tried but failed, because there're dots in the variable name which are invalid